js/calculate.js

```diff
@@ -0,0 +1,138 @@
+const operate = require('./operate');
+
+function calculate(input, calcObject) {
+  let { operandOne = null, operandTwo = null, displayValue = null, operator = null } = calcObject;
+  if (
+    input === '0' ||
+    input === '1' ||
+    input === '2' ||
+    input === '3' ||
+    input === '4' ||
+    input === '5' ||
+    input === '6' ||
+    input === '7' ||
+    input === '8' ||
+    input === '9'
+  ) {
+    if (operandOne === null) {
+      operandOne = input;
+      displayValue = operandOne;
+    } else if (operator === null) {
+      operandOne = `${operandOne}${input}`;
+      displayValue = operandOne;
+    } else if (operandTwo === null) {
+      operandTwo = input;
+      displayValue = operandTwo;
```

Comment:
@Yinkusadel Thank you for the review but you should be going through the code more thoroughly if you check the if statement that I first created I already said that the input should be from 0 - 9 that is whatever input the user wishes to click will be the number, And also you asked when operandTwo is null what operandTwo is to be displayed at first when it's null?? When the opearandTwo is null it will display the input and the input is the number which is 0 - 9 when operandTwo is input I want it to display what is inside the opearandTwo

Reply:
@Yinkusadel `operandTwo` is already assigned a value of `input` so it's no more null. @moshoodabdullahi maybe you should have just assigned both of them the `input` value